src/main/java/frc/robot/Constants.java

```diff
@@ -277,6 +279,46 @@ private enum Mk4iReductions {
         }
     }
 
+
+    /**
+     * Elevator Constants
+     */
+
+    public static final class Elevator {
+        public static final int RIGHT_ID = 7;
+        public static final int LEFT_ID = 4;
+        public static final int LIMIT_ID = 0;
+
+        public static final NeutralModeValue BREAK = NeutralModeValue.Brake;
+
+        // PID and feedforward
+        public static final double KP = 5.0;
+        public static final double KI = 1.5;
+        public static final double KD = 0.0;
+        public static final double KS = 0.1675;
+        public static final double KV = 0.0;
+        public static final double KA = 0.0;
+        public static final double KG = 0.3375;
+        public static final AngularVelocity MAX_VELOCITY = RotationsPerSecond.of(0.0);
+
+        // positions
+        public static final Distance HOME = Inches.of(2);
+        public static final Distance P0 = Inches.of(60);
+        public static final Distance P1 = Inches.of(32);
+        public static final Distance P2 = Inches.of(44);
+        public static final Distance P3 = Inches.of(48);
+        public static final Distance P4 = Inches.of(72);
+
+
+        public static final double gearRatio = 1.0;
+        public static final double SensorToMechanismRatio =
+            Constants.Elevator.ROTATIONS_AT_TOP.in(Rotations)
+                / Constants.Elevator.METERS_AT_TOP.in(Meters);
+        public static final Distance METERS_AT_TOP = Meters.of(Units.inchesToMeters(72.0));
```

Comment:
No point in converting inches to meters and then back to a unit. Just do `Inches.of(XX)`